Dependencies.cmake

```diff
@@ -0,0 +1,52 @@
+# Preserve build options.
```

Comment:
I don't think we do cmake for our automatic license header. But in case we do that in the future, let's put this in the exception as well.

https://github.com/NVIDIA/Fuser/blob/main/tools/update_copyright.py#L25